utils_cv/multi_object_tracking/display_with_bb.py

```diff
@@ -0,0 +1,123 @@
+import cv2
```

Comment:
Can you run flake8 and black please on this (and other files) to make sure formatting looks good. Will review then, at the moment it's hard to read.

See for some more info here. If this pre-commit doesn't setup correctly (I am having problems on windows DSVMs) then can also just run manually e.g. "flake8 <filepath>"
https://github.com/microsoft/computervision-recipes/blob/master/CONTRIBUTING.md#steps-to-contributing